app/config/constants.js

```diff
@@ -136,4 +136,7 @@ export default {
     USER_PAGEVIEW: 'Pageview (per rol)',
     FIRST_ACTION: 'Eerste actie',
   },
+  DEFAULTS: {
+    FALLBACK_EXTENSION : 'pdf',
+  }
```

Comment:
In the general scheme of this file this is not a good naming strategy.
`DEFAULTS` is very vague as to what you need defaults for.
Something like this would make more sense.
```
  FILE_EXTENSIONS: {
    FALLBACK : 'pdf',
    PDF: 'pdf',
    DOCX: 'docx',
  }
```